Address-Finder/Address-Finder/ViewController.swift

```diff
@@ -6,21 +6,170 @@
+//        let parameters: [String: Any] = [ "query": keyword ]
+//        Alamofire.request("https://dapi.kakao.com/v2/local/search/address.json",
+//                          method: .get,
+//                          parameters: parameters,
+//                          headers: headers)
+//            .responseJSON(completionHandler: { response in
+//                debugPrint(response)
+//                switch response.result {
+//                case .success(let value):
+//                    if let addressList = JSON(value)["documents"].array {
+//                        for item in addressList{
+//                            let streetAddr = item["road_address"]["address_name"].string ?? ""
+//                            //let Addr = JSON(value)["address"]
+//                            //let jibunAddr = item["zip_code"].string ?? ""
+//
+//                            let jibunAddr = item["address"]["address_name"].string ?? ""
+//                            self.resultList.append(Address(roadAddress: streetAddr, jibunAddress: jibunAddr))
+//                        }
+//                    }
+//                    self.resultTableView.reloadData()
+//                case .failure(_):
+//                    debugPrint(Error.self)
+//                }
+//        })
+//    }
+}
+
+extension ViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return resultList.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath) as! AddressTableCell
+        let roadAddress = checkNil(resultList[indexPath.row].roadAddress?.addressName)
+        let jibunAddress = checkNil(resultList[indexPath.row].address?.addressName)
         
-        let SearchController = UISearchController(searchResultsController: nil)
-        SearchController.searchBar.placeholder = "Search Address"
-        self.navigationItem.searchController = SearchController
+        cell.roadAddressLabel.text = ("도로명: ") + roadAddress
+        cell.jibunAddressLabel.text = ("지번: ") + jibunAddress
+        //print(resultList)
+        return cell
     }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return view.frame.height / 10
+    }
+}
 
+extension ViewController: UITableViewDelegate {
 
 }
 
+extension ViewController: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        doSearchAddress(keyword: checkNil(textField.text))
+        return true
+    }
+}
+
+extension UIViewController {
+    func hideKeyboard(){
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self,
+            action: #selector(UIViewController.dismissKeyboard))
```

Comment:
```suggestion
            action: #selector(dismissKeyboard))
```
UIViewController 는 없어도 될 것 같은데 왜 있을까요??